config/database.yml

```diff
@@ -1,30 +1,23 @@
-
 default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-
-development:
-  <<: *default
-  adapter: postgresql
-  encoding: unicode
-  database: wbooks
-  host: localhost
-  username: wbooks-api
-  password: wbooks-api
-
-test:
-  <<: *default
```

Comment:
you may extend test from development instead.

Reply:
Done.